arrow-site/docs/_data/sidebar-core.yml

```diff
@@ -36,6 +36,25 @@ options:
       - title: Extensions
         url: /apidocs/arrow-core/arrow.core/index.html#functions
 
+  - title: Continuations and programs
+
+    nested_options:
+
+      - title: arrow.core.continutions.Effect
+        url: /apidocs/arrow-core/arrow.core.continuations/-effect/
```

Comment:
Yes sounds great too, it would be great if we could link EffectScope with EagerEffectScope and showing they cover similar signatures. 
And rearranging it sounds super. 

On that note now that we moved on from Data type + extension (aka implementation of a Type class) we might want to consider just having Data types and computations. 

The few typeclasses we have left can also have a dedicated place, if necessary 
Just as a note, though.